### PR TITLE
Encodings

### PR DESCRIPTION
**Pull request by gmponos**
I am from Greece. So most of my data are in Greek language inside my DB and they are stored as UTF-8. The data can be viewed normally if you open the with a text editor like notepad++ but If I need them to be viewed with MS Excel they will be scrambled.

This way you can add support for different encoding.

### CODE REVIEW

File: src/View/CsvView.php [hunk 1]
```diff
@@ -373,6 +383,12 @@ protected function _generateRow($row = null)
             $csv = str_replace("\n", $eol, $csv);
         }
 
+        $dataEncoding = $this->viewVars['_dataEncoding'];
+        $csvEncoding = $this->viewVars['_csvEncoding'];
+        if ($dataEncoding != $csvEncoding) {
```
Comment (dereuromark) [inline src/View/CsvView.php]: should be strict `!==`

Comment (gmponos) [inline src/View/CsvView.php]: Can you give me some help ho to update this? Shall I push my change again and ask for a PR once more?

Comment (dereuromark) [inline src/View/CsvView.php]: You can directly push a commit on top, the PR will automatically be updated.

**Comment by josegonzalez** [comment]
Thanks for the pr!

**Pull request merged**